uk_ua.lang

```diff
@@ -1,154 +1,578 @@
+gui.xaero_pac_expired_claim_tooltip=Заява сплинула%1$s
+gui.xaero_pac_claims=Відбивання претензій до чанків
+gui.xaero_box_pac_claims=Відображення претензій до чанків з моду Open Parties and Claims у вигляді кольорового накладання.
+gui.xaero_pac_claims_border_opacity=Непрозорість межі претензій до чанків.
+gui.xaero_box_pac_claims_border_opacity=Рівень прозорості межі кольорового накладання претензій до чанків. Для відображення претензій до чанків необхідно встановити мод Open Parties and Claims.
+gui.xaero_pac_claims_fill_opacity=Непрозорість заповнення претензій до чанків.
+gui.xaero_box_pac_claims_fill_opacity=Рівень прозорості кольорового заповнення претензій до чанків. Для відображення претензій до чанків необхідно встановити мод Open Parties and Claims.
+gui.xaero_pac_current_claim=Відображати поточне володіння
+gui.xaero_box_pac_current_claim=Відображати інформацію про володіння у чанку, в якому ви перебуваєте. Володіння чанками потребує Open Parties and Claims мод.
+gui.xaero_category_include_list_include_in_super=Включити до супер-категорії
+gui.xaero_box_category_include_list_include_in_super=Автоматично включати всі сутності з цього списку та інших списків, включених до цієї категорії, до суперкатегорії/вищої категорії, окрім сутностей зі списку винятків. Таким чином, вам не доведеться вручну додавати дублікати на кожному рівні.
+gui.xaero_temp_waypoints_global=Тимчасові глобальні маркери
+gui.xaero_box_temp_waypoints_global=Коли увімкнено, тимчасові маркери розглядаються як глобальні та ігнорують обмеження на їх відображення.
+gui.xaero_keep_enlarged_minimap_unlocked=Зберігати напрямок при збільшення
+gui.xaero_box_keep_enlarged_minimap_unlocked=Якщо можливо, не закріплюйте мінімапу на північ при збільшенні, що можливо лише при використанні рамки у формі кола на момент написання цього опису.
+gui.xaero_enlarged_minimap_a_toggle=Збільшення мінімапи увімкнено
+gui.xaero_box_enlarged_minimap_a_toggle=Використовувати клавішу "Збільшити мінімапу" як перемикач замість утримання.
+gui.xaero_waypoint_visibility_type_world_map_local= Лок. мапа світу
+gui.xaero_waypoint_visibility_type_world_map_global=Глоб. мапа світу
+gui.xaero_box_visibility_type=Тип видимості у грі/на мапі \n \n Локальний - видимий лише в межах максимальної дальності відображення маркера \n Глобальний - завжди видимий \n Лок. мапа світу - не видима, але вважається локальною \n Глобальна мапа світу. - невидимий, але вважається глобальним
+gui.xaero_option_requires_ingame=Ця опція вимагає щоб ви були у грі.
+gui.xaero_alternative_list_players=Альтернатива "Список гравців"
+gui.xaero_display_stained_glass=Від. пофарбованого скла
+gui.xaero_waypoint_onmap_scale=Масштаб маркерів на мапі
+gui.xaero_waypoint_onmap_scale_auto=Авто
+gui.xaero_switch_to_auto_on_death=Авто. маркер при смерті
+gui.xaero_box_switch_to_auto_on_death=Чи перемикати позначення на авто. при створенні точки смерті. Це не впливає на місце створення точки смерті.
```

Comment:
This is not about where the deathpoint is created. It's the tooltip for the line 535.

Reply:
Do not understand the context here at all. Neither on eng nor ukr versions.